### PR TITLE
p4-881 prefactor - re-factor tests

### PR DESCRIPTION
**Pull request by starswan**

### CODE REVIEW

File: spec/services/personal_care_needs/importer_spec.rb [hunk 1]
```diff
@@ -7,21 +7,21 @@
...
-        end_date: '2010-06-21',
-      },
+        {
+            problem_type: 'MATSTAT',
+            problem_code: 'ACCU9',
+            problem_status: 'ON',
+            problem_description: 'Preg, acc under 9mths',
+            start_date: '2010-06-21',
+            end_date: '2010-06-21',
+        },
```
Comment (spikeheap) [inline spec/services/personal_care_needs/importer_spec.rb]: Hi @starswan it looks like you've got your indentation set to 4 spaces. There are a few places in this PR which could do with being consistent with 2 space tabs

File: spec/services/alerts/importer_spec.rb [hunk 2]
```diff
@@ -42,56 +44,74 @@
...
+    [{
+      person_nomis_prison_number: person.nomis_prison_number,
+      from_location_nomis_agency_id: 'BXI',
+      to_location_nomis_agency_id: 'WDGRCC',
+      date: '2019-08-19',
+      time_due: '2019-08-19T17:00:00',
+      status: 'requested',
+      nomis_event_id: 123_456,
+    }]
+  end
```
Comment (spikeheap) [inline spec/services/alerts/importer_spec.rb]: Is `moves` used in this test? I'm probably just being unobservant...

**Comment by spikeheap** [comment]
The rubocop comments seem valid:

```
spec/services/integration/alerts_importer_spec.rb:5:1: C: RSpec/FilePath: Spec path should end with moves/importer*_spec.rb. (http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)
RSpec.describe Moves::Importer do
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/services/integration/people_importer_spec.rb:5:1: C: RSpec/FilePath: Spec path should end with moves/importer*_spec.rb. (http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)
RSpec.describe Moves::Importer do
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/services/integration/personal_care_needs_importer_spec.rb:5:1: C: RSpec/FilePath: Spec path should end with moves/importer*_spec.rb. (http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)
RSpec.describe Moves::Importer do
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Perhaps `integration/personal_care_needs_importer_spec` could become `integration/moves/importer_personal_care_needs_spec` (or something) ?

File: spec/services/personal_care_needs/importer_spec.rb (hunk 1 above)
Comment (starswan) [reply spec/services/personal_care_needs/importer_spec.rb]: you just beat me to it...:-)

**Pull request merged**